src/_provider/MyAnimeList_api/single.ts

```diff
@@ -72,6 +74,37 @@ export class Single extends SingleAbstract {
     }
   }
 
+  _getStartDate() {
+    return this.animeInfo.my_list_status.start_date;
+  }
+
+  _setStartDate(startDate) {
+    this.animeInfo.my_list_status.start_date = startDate;
+  }
+
+  _getFinishDate() {
+    return this.animeInfo.my_list_status.finish_date;
+  }
+
+  _setFinishDate(finishDate) {
+    this.animeInfo.my_list_status.finish_date = finishDate;
+  }
+
+  _getRewatchCount() {
+    if (this.type === 'manga') {
+      return this.animeInfo.my_list_status.num_times_reread;
+    }
+    return this.animeInfo.my_list_status.num_times_rewatched;
+  }
+
+  _setRewatchCount(rewatchCount) {
```

Comment:
I could always just add `Math.min(rewatchCount, 255)`, but it wouldn't fix the list sync page always showing it in that case if MAL is set as a slave. Ex. if AniList as a master has 300 it will always say MAL is out of sync, but at least the sync itself will be successful

Reply:
Ah got it. I would agree then with not handling it